repotracker/repotracker_test.go

```diff
@@ -761,15 +756,23 @@ func TestBuildBreakSubscriptions(t *testing.T) {
 		},
 	}
 	assert.NoError(u4.Insert())
+	assert.NoError(AddBuildBreakSubscriptions(&v1, &proj2))
+	assert.NoError(db.FindAllQ(event.SubscriptionsCollection, db.Q{}, &subs))
+	assert.Len(subs, 2)
+
+	// project has it enabled, but user doesn't want notifications
+	subs = []event.Subscription{}
+	assert.NoError(db.Clear(event.SubscriptionsCollection))
 	v3 := model.Version{
 		Id:         "v3",
-		Identifier: proj1.Id,
+		Identifier: proj2.Id,
 		Requester:  evergreen.RepotrackerVersionRequester,
 		Branch:     "branch",
 		AuthorID:   u4.Id,
 	}
 	assert.NoError(AddBuildBreakSubscriptions(&v3, &proj2))
 	assert.NoError(db.FindAllQ(event.SubscriptionsCollection, db.Q{}, &subs))
+	assert.EqualValues(utility.ToStringPtr("HELL0173"), subs[0].Subscriber.Target)
```

Comment:
Now that I'm looking at how the `interface{}` stores the string, it seems like kind of a weird comparison since this should ideally compare strings and not pointer addresses. If it's stored as a pointer in the `interface{}`, can we do a type assert to a string pointer `subs[0].Subscriber.Target.(string)` and then `utility.FromStringPtr` it?